frontend/src/components/PathAppLayout.module.css

```diff
@@ -0,0 +1,83 @@
+.pathwayApplicationBase {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.applicationContainer {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 100%;
+  box-shadow: 0 5px 10px rgba(0, 0, 0, 0.25);
+}
+
+.title {
+  text-align: center;
+  font: var(--font-title);
+  padding-bottom: 2rem;
+  line-height: 48px;
+  letter-spacing: 0.02em;
+}
+
+.pathContent {
+  text-align: center;
+  font-family: var(--font-body);
+  width: 75em;
+}
+
+.centeredContainer {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding-bottom: 3em;
+  padding-top: 3em;
+}
+
+.note {
+  display: flex;
+  justify-content: center;
+  text-decoration: underline;
+  text-align: center;
+  padding-bottom: 4em;
+}
+
+.red {
+  color: var(--color-ccidc-red);
+}
+
+.backArrow img:hover {
+  content: url("../assets/backArrowHovered.svg");
+}
+
+.arrow img:hover {
+  content: url("../assets/arrowHovered.svg");
+}
+
+.navigationContainer {
+  height: 6rem;
+  width: 926px;
+  padding: 10px;
+  box-shadow: 0 5px 10px rgba(0, 0, 0, 0.1);
+  display: flex;
+  justify-content: space-between;
+}
+
+.arrow {
+  margin-right: 5rem;
+}
+
+.backArrow {
+  margin-left: 5rem;
+}
+
+button img {
```

Comment:
Let's try to give the button's that this style is trying to refer to a classname as opposed to selecting the button element. Just as a precaution as this could affect the stylings of other buttons on the page even if they are not in this component. 

Though great use of the selector of `button img`. the img won't need a classname. Just replace button with a slightly more unique classname so we don't clash with other classnames as much